### PR TITLE
Lro Wac fix

### PR DESCRIPTION
**Pull request by acpaquette**
Updates the LRO Wac driver and push frame timing.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: ale/base/type_sensor.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def ephemeris_time(self):
           ephemeris times split based on image lines
         """
 
-        return np.arange(self.ephemeris_start_time + (.5 * self.exposure_duration), self.ephemeris_stop_time + self.interframe_delay, self.interframe_delay)
+        return np.arange(self.ephemeris_start_time, self.ephemeris_stop_time, self.interframe_delay)
```
Comment (Kelvinrr) [inline ale/base/type_sensor.py:118]: I imagine this was just changed to match ISIS?

File: ale/base/type_sensor.py [hunk 2]
```diff
@@ -162,7 +162,7 @@ def ephemeris_stop_time(self):
         : double
           Center ephemeris time for an image
         """
-        return self.ephemeris_start_time + (self.interframe_delay) * (self.num_frames - 1) + self.exposure_duration
+        return self.ephemeris_start_time + (self.interframe_delay * self.num_frames)
```
Comment (Kelvinrr) [inline ale/base/type_sensor.py:165]: Same here

File: ale/drivers/lro_drivers.py [hunk 3]
```diff
@@ -1192,7 +1191,7 @@ def num_frames(self):
         : int
           Number of frames in the image
         """
-        return self.image_lines // (self.framelet_height // self.sampling_factor)
+        return (self.image_lines // (self.framelet_height // self.sampling_factor)) + 1
```
Comment (Kelvinrr) [inline ale/drivers/lro_drivers.py:1194]: Why did it need a +1?

File: ale/base/type_sensor.py (hunk 1 above)
Comment (acpaquette) [reply ale/base/type_sensor.py:118]: Somewhat, rather than apply the `+ (.5 * self.exposure_duration)` in the pushframe sensor type, the start time should be updated on the driver.

File: ale/base/type_sensor.py (hunk 2 above)
Comment (acpaquette) [reply ale/base/type_sensor.py:165]: Yes, the math here was weird relative to ISIS

File: ale/drivers/lro_drivers.py (hunk 3 above)
Comment (acpaquette) [reply ale/drivers/lro_drivers.py:1194]: This was also an ISIS motivated change, the LroWideAngleCamera in ISIS calculated one extra frame. I am not sure why as the PushFrameCameraDetectorMap logic is hard to follow

**Review by Kelvinrr — APPROVED** [review]
